README.md

```diff
@@ -109,8 +118,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
-
+FSMdiagram.jpg
```

Comment:
Love the hairy circles :)

Good job noticing that you can loop back to S1 on adjectives.